Update this page to include links and FAQ (#1693)
SumoLogic/sumologic-documentation: docs/integrations/community-ecosystem-apps.md

* [Palo Alto Cortex XDR](https://github.com/SumoLogic/sumologic-content/tree/master/Palo_Alto_Networks/Cortex_XDR)
* [Proofpoint Targeted Attack Protection (TAP)](https://github.com/SumoLogic/sumologic-content/tree/master/Proofpoint/Proofpoint_TAP)
* [Symantec Web Security Service (WSS)](https://github.com/SumoLogic/sumologic-content/tree/master/Symantec/WSS)
* [Sumo Logic Traces](https://github.com/SumoLogic/sumologic-content/tree/master/Sumo_Logic_Native/Traces)
* [Tenable.io](https://github.com/SumoLogic/sumologic-content/tree/master/Tenable/Tenable_IO)
* [Qualys VMDR & Inventory](https://github.com/SumoLogic/sumologic-content/tree/master/Qualys)

## Contributing Guidelines

Content can include vendor-based/custom applications, dashboards, search queries, field extraction rules (FERs), tooling, etc. Please complete each of the required fields in the [Sumo Logic Community Content Submission](https://forms.gle/KQBLBuMuUw85xtRi9) form, and follow the guidelines below:
- Verify that all sensitive data has been removed or redacted BEFORE submitting content. This includes PII, credentials, network data, etc. If you provided screenshots, either blur or mark over this sensitive data.

> - All application, dashboard, and search content should be in JSON format (exported from Sumo Logic). Please use a descriptive naming convention for files (Company-Product-Function. Ex: AWS-Kinesis-Errors.json).

Reviewer: ```suggestion
- All application, dashboard, and search content should be in JSON format (exported from Sumo Logic). Please use a descriptive naming convention for files, preferably Company-Product-Function (example: **aws-kinesis-errors.json**).
```

- When possible, include a screenshot of your dashboards in .png or .jpg format. Name your screenshots the same as their respective JSON content.
- If you have notes on how to collect data for specific content, please provide those steps as well.
- If you have a GitHub Repository for your content already, please feel free to provide the link to this repository, and make sure it has Public access.

Once submitted, Sumo Logic will review your content for publishing to our Community Ecosystem repository.

### Alternative Submission

Follow the "fork-and-pull" Git workflow:
- **Fork** the repo on GitHub.
- **Clone** the project to your own machine.
- **Create** any new folders/subfolders necessary for your content.
   - Add your content to these folders.
   - Verify that all sensitive data has been removed or redacted BEFORE submitting a PR. This includes PII, Credentials, and Network data. Screenshots included (either blur or mark over).

>    - All application, dashboard and search content should be in JSON format (exported from Sumo Logic). Please use a descriptive name, ideally the naming convention <company>-<product>-<function> (example: AWS-Kinesis-Errors.json).

Reviewer: ```suggestion
   - All application, dashboard and search content should be in JSON format (exported from Sumo Logic). Please use a descriptive name, ideally the naming convention Company-Product-Function (example: **aws-kinesis-errors.json**).
```

More information on this initiative and how to contribute can be found [here](#faq).

## Community Content Catalog

> Community content is stored in our [Community Content Repository](https://github.com/SumoLogic/sumologic-content). Please visit the following links to access related content:

Reviewer: @wjakelee , do you think this section might look better as a bulleted list?

Also, thanks for putting this together!

Author: Yeah I think you are correct. Probably better as bulleted list!

> [EC2 Host Metrics for OpenTelemetry](https://github.com/SumoLogic/sumologic-content/tree/master/Amazon_Web_Services/EC2/Host_Metrics_OTEL)
> 
> [Fortinet Fortigate Firewall](https://github.com/SumoLogic/sumologic-content/tree/master/Fortinet/Fortigate_Firewall)
> 
> [GCP Loud Balancer Metrics](https://github.com/SumoLogic/sumologic-content/tree/master/GCP/Load_Balancer_Metrics)
> 
> [JAMF](https://github.com/SumoLogic/sumologic-content/tree/master/JAMF)
> 
> [Palo Alto Cortex XDR](https://github.com/SumoLogic/sumologic-content/tree/master/Palo_Alto_Networks/Cortex_XDR)
> 
> [Proofpoint Targeted Attack Protection (TAP)](https://github.com/SumoLogic/sumologic-content/tree/master/Proofpoint/Proofpoint_TAP)
> 
> [Symantec Web Security Service (WSS)](https://github.com/SumoLogic/sumologic-content/tree/master/Symantec/WSS)
> 
> [Sumo Logic Traces](https://github.com/SumoLogic/sumologic-content/tree/master/Sumo_Logic_Native/Traces)
> 
> [Tenable.io](https://github.com/SumoLogic/sumologic-content/tree/master/Tenable/Tenable_IO)
> 
> [Qualys VMDR & Inventory](https://github.com/SumoLogic/sumologic-content/tree/master/Qualys)

Reviewer: ```suggestion
* [EC2 Host Metrics for OpenTelemetry](https://github.com/SumoLogic/sumologic-content/tree/master/Amazon_Web_Services/EC2/Host_Metrics_OTEL)
* [Fortinet Fortigate Firewall](https://github.com/SumoLogic/sumologic-content/tree/master/Fortinet/Fortigate_Firewall)
* [GCP Loud Balancer Metrics](https://github.com/SumoLogic/sumologic-content/tree/master/GCP/Load_Balancer_Metrics)
* [JAMF](https://github.com/SumoLogic/sumologic-content/tree/master/JAMF)
* [Palo Alto Cortex XDR](https://github.com/SumoLogic/sumologic-content/tree/master/Palo_Alto_Networks/Cortex_XDR)
* [Proofpoint Targeted Attack Protection (TAP)](https://github.com/SumoLogic/sumologic-content/tree/master/Proofpoint/Proofpoint_TAP)
* [Symantec Web Security Service (WSS)](https://github.com/SumoLogic/sumologic-content/tree/master/Symantec/WSS)
* [Sumo Logic Traces](https://github.com/SumoLogic/sumologic-content/tree/master/Sumo_Logic_Native/Traces)
* [Tenable.io](https://github.com/SumoLogic/sumologic-content/tree/master/Tenable/Tenable_IO)
* [Qualys VMDR & Inventory](https://github.com/SumoLogic/sumologic-content/tree/master/Qualys)
```

## Contributing Guidelines

Content can include vendor-based/custom applications, dashboards, search queries, field extraction rules (FERs), tooling, etc. Please complete each of the required fields in the [Sumo Logic Community Content Submission](https://forms.gle/KQBLBuMuUw85xtRi9) form, and follow the guidelines below:

> - Verify that all sensitive data has been removed or redacted BEFORE submitting content. This includes PII, Credentials, Network data, etc. If you provided screenshots, either blur or mark over this sensitive data.

Reviewer: ```suggestion
- Verify that all sensitive data has been removed or redacted BEFORE submitting content. This includes PII, credentials, network data, etc. If you provided screenshots, either blur or mark over this sensitive data.
```

- All application, dashboard, and search content should be in JSON format (exported from Sumo Logic). Please use a descriptive naming convention for files (Company-Product-Function. Ex: AWS-Kinesis-Errors.json).
- When possible, include a screenshot of your dashboards in .png or .jpg format. Name your screenshots the same as their respective JSON content.
- If you have notes on how to collect data for specific content, please provide those steps as well.
- If you have a GitHub Repository for your content already, please feel free to provide the link to this repository, and make sure it has Public access.

Once submitted, Sumo Logic will review your content for publishing to our Community Ecosystem repository.

### Alternative Submission

> Follow the "fork-and-pull" Git workflow:
> - **Fork** the repo on GitHub
> - **Clone** the project to your own machine
> - **Create** any new folders/subfolders necessary for your content
> 	- Add your content to these folders
> 	- Verify that all sensitive data has been removed  or redacted BEFORE submitting a PR. This includes PII, Credentials, and Network data. Screenshots included (either blur or mark over).
> 	- All application, dashboard and search content should be in JSON format (exported from Sumo Logic). Please use a descriptive naming convention (Company-Product-Function. Ex: AWS-Kinesis-Errors.json)
> 	- When possible, include a screenshot of your dashboards in .png or .jpg format. Name your screenshots the same as their respective JSON content. If there is more than one, please place these images in a Screenshots folder.
> 	- Create README.md file within each subfolder to track descriptions of the app, guidance on setting up the data collection, authors, versions, dates, and links to 3rd party docs
> - **Commit** the changes on your local machine to your own fork/branch
> - **Push** your work back up to your fork
> - **Submit** a pull request so that we can review your changes and publish

Reviewer: ```suggestion
Follow the "fork-and-pull" Git workflow:
- **Fork** the repo on GitHub.
- **Clone** the project to your own machine.
- **Create** any new folders/subfolders necessary for your content.
   - Add your content to these folders.
   - Verify that all sensitive data has been removed or redacted BEFORE submitting a PR. This includes PII, Credentials, and Network data. Screenshots included (either blur or mark over).
   - All application, dashboard and search content should be in JSON format (exported from Sumo Logic). Please use a descriptive name, ideally the naming convention <company>-<product>-<function> (example: AWS-Kinesis-Errors.json).
   - When possible, include a screenshot of your dashboards in .png or .jpg format. Name your screenshots the same as their respective JSON content. If there is more than one, please place these images in a Screenshots folder.
   - Create README.md file within each subfolder to track descriptions of the app, guidance on setting up the data collection, authors, versions, dates, and links to third party docs.
- **Commit** the changes on your local machine to your own fork/branch.
- **Push** your work back up to your fork.
- **Submit** a pull request so that we can review your changes and publish.
```

## FAQ

> Sumo Logic Community Ecosystem Apps are provided and supported by our internal and external user and creator community. Generally these apps should be viewed as open source content, and we strive to ensure that they are proactively reviewed by our community to ensure usability, use case coverage, and technical quality.

Reviewer: ```suggestion
Sumo Logic Community Ecosystem Apps are provided and supported by our internal and external user and creator community. Generally, these apps should be viewed as open source content, and we strive to ensure that they are proactively reviewed by our community to ensure usability, use case coverage, and technical quality.
```

### What is this?

A central place to list community content from the internal and external Sumo Community to enable more rapid time to value for customers.

### How do I use this content?

> Methods to using each piece of content are outlined their corresponding locations. For content like Apps, Log Searches, and FERs, you can use Sumo Logic's [import](https://help.sumologic.com/docs/get-started/library/#import-content) feature to add this content to your Sumo Logic Library.

Reviewer: ```suggestion
Methods to using each piece of content are outlined their corresponding locations. For content like Apps, Log Searches, and FERs, you can use Sumo Logic's [import](/docs/get-started/library/#import-content) feature to add this content to your Sumo Logic Library.
```
